lib/MusicBrainz/Server/Edit/ReleaseGroup/Create.pm

```diff
@@ -59,7 +60,8 @@ sub build_display_data
         artist_credit => to_json_object(artist_credit_preview($loaded, $self->data->{artist_credit})),
         name          => $self->data->{name} || '',
         comment       => $self->data->{comment} || '',
-        type          => $type ? to_json_object($loaded->{ReleaseGroupType}{$type}) : undef,
+        # Older edits store historic (pre-split) "secondary" types
+        type          => get_historic_type($type, $loaded),
```

Comment:
What if we had `get_historic_type` return whether it's a historic type, and add something like `has_historic_type` to the display data?

Reply:
So, have get_historic_type return ($type, $has_historic_type), and then use that? That could work. What'd be the best way of doing that assignment inside the map at https://github.com/metabrainz/musicbrainz-server/pull/2138/files#diff-50d2fc9c2cb554797610757d6cff4d398976e3022b4aca65265ff2ed6cfe4284R59-R70 without just calling it twice?

Reply:
Maybe adding a `historic` attribute to `Entity::ReleaseGroupType` would be simpler.

Reply:
Made that change, it does seem simple.